development/webpack/test/webpack.config.test.ts

```diff
@@ -231,7 +233,7 @@ ${Object.entries(env)
     assert.deepStrictEqual(manifestPlugin.options.description, null);
     assert.deepStrictEqual(manifestPlugin.options.zip, true);
     assert(manifestPlugin.options.zipOptions, 'Zip options should be present');
-    assert.strictEqual(manifestPlugin.options.transform, undefined);
+    assert.notEqual(manifestPlugin.options.transform, undefined);
```

Comment:
Note: This is because now we have enabled `test` in args, then the previous logic will add this function `addTabsPermission`:  
```
if (args.test) {
    // test builds need "tabs" permission for switchToWindowWithTitle
    transforms.push(addTabsPermission);
  }
```